cmake/python.cmake

```diff
@@ -9,6 +9,10 @@
 # https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/python.cmake
 # 
 
+# set the path to the package root based on the location of this file
+set(mpi_cmake_modules_SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/..)
```

Comment:
I would not put it here. I would create a `export.cmake` (or any similar name) with the intend to export some variable specific to this package. Like for example:

- `mpi_cmake_modules_SOURCE_DIR`
- `mpi_cmake_modules_RESSOURCE_DIR = ${mpi_cmake_modules_SOURCE_DIR}/ressource/`
- `mpi_cmake_modules_CMAKE_DIR = ${mpi_cmake_modules_SOURCE_DIR}/cmake/`

WDYT?

Reply:
I think that's a good idea :+1:.  What is the supposed usage of this file?  Would it simply be included in the `python.cmake` instead of the `set` or is there some smarter way?

Reply:
You can include it from the python.cmake yes. At least that is how I see it.

Did you also check the behavior of doxygen.cmake? it is not the prettiest but I think it works on catkin_tools.

Reply:
When I mean include I litterally mean include:
` include(ressource/export.cmake)`

Reply:
Alright.  I didn't test doxygen yet but already saw that it uses the same variable. So probably needs the same fix.  I'll take care of it and update here.

Reply:
done